app/controllers/evidences_controller.rb

```diff
@@ -1,14 +1,23 @@
 # frozen_string_literal: true
 
 class EvidencesController < ApplicationController
+  before_action :set_media
+
   def destroy
-    media = ActiveStorage::Attachment.find(params[:media_id])
-    media.purge if media.record.errors.empty?
+    if @media.record.errors.empty?
```

Comment:
I think you can use @media.record.valid? and it does the same thing

Reply:
Thanks for the recommendation, I will consider it.